simtools/applications/validate_camera_efficiency.py

```diff
@@ -23,6 +23,14 @@
         Telescope model name (e.g. LST-1, SST-D, ...)
     model_version (str, optional)
         Model version (default='Current')
+    zenith_angle (float, optional)
+        Zenith angle in degrees (between 0 and 180) (default=20).
+    azimuth_angle (float, optional)
+        Telescope pointing direction in azimuth. It can be in degrees between 0 and 360 or
+        one of north, south, east or west (case insensitive). Note that North is 0 degrees
+        and the azimuth grows clockwise, so East is 90 degrees (default=0).
+    nsb_spectrum (str, optional)
+        File with NSB spectrum to use (following sim_telarray required format).
```

Comment:
I would not spare on the explanation of this file. Where can we find an example? Could you give an example of it and/or its format?